barberia/src/App/pages/Congratulation.js

```diff
@@ -4,13 +4,26 @@ import "./congratulation.css";
 
 function congratulation({ ...props }) {
   const userData = JSON.parse(localStorage.getItem("confirmationData"));
-  const { email, phone } = userData;
+  const { email, phone, name } = userData;
 
-  const moveTo = destination => {
-    window.location.assign(destination);
+  // console.log(JSON.stringify(userData));
```

Comment:
Do we need the console.log in a working piece of code that you want to merge into master?